quantum/keymap_extras/keymap_french.h

```diff
@@ -146,6 +146,7 @@
 #define FR_PIPE ALGR(FR_MINS) // |
 #define FR_GRV  ALGR(FR_EGRV) // ` (dead)
 #define FR_BSLS ALGR(FR_UNDS) // (backslash)
+#define FR_CIRC ALGR(FR_CCED) // ^
```

Comment:
We should give him another name, FR_CIRC is already used in many places and devs except to be the `dead ^`.

Reply:
`FR_CIRC` would be the dead `^`, unless there is a non-dead equivalent present in the layout (as there is here), in which case the dead key is `FR_DCIR`.

Reply:
We can make the PR breaking changes with this changes:

```
#define FR_DCIR KC_LBRC
#define FR_CIRC ALGR(FR_CCED)
```

If we do this changes, should this PR migrate keymaps in the QMK repo?

Reply:
I can only find one instance in the repo that isn't a default keymap and isn't using the macOS version, `layouts/community/ergodox/french_hacker/keymap.c`. If we get signoff from the contributor of that layout, it doesn't have to go through breaking changes.